README.md

```diff
@@ -1,24 +1,25 @@
-## Daedalus Dock codebase
+# Dionysus Dock codebase
```

Comment:
Kinda not cool to ape all of the greek theming and keep the same initialism as my server. I'd very much appreciate not doing that, given that I expressly said I did not want my code involved with an ERP server, and do not want people conflating the two.